tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Italics: getDownloadUrl() &gt; <em>getDownloadUrl()</em></li><li>I recommend adding a note that for large or long-running exports, using the <code>ee.batch.Export</code>  module (like described above) is a better method.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Done</p>